### PR TITLE
Added share button to RequestEpisode component and adjust styles

### PR DESCRIPTION
**Pull request by suryapratapsingh26**
a "Share" button was added to the RequestEpisode.jsx component. This button allows users to easily share their favorite episodes, thereby enhancing user engagement. Additionally, adjustments were made to the styles in the newly created RequestEpisode.css file to ensure proper alignment and appearance of the button.

https://github.com/Avdhesh-Varshney/Chanakya-Niti/assets/146753809/8b73d415-8028-429e-833a-35e756d478d8

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@suryapratapsingh26 LGTM. But,
- Resolve conflicts.
- Also properly linked your issue.

**Comment by suryapratapsingh26** [comment]
RESOLVEissue #19 For adding feature for sharing specific episodes #19

**Comment by suryapratapsingh26** [comment]
> @suryapratapsingh26 LGTM. But,
> 
> * Resolve conflicts.        I have resolved the conflicts review it
> * Also properly linked your      I have linked issue also

File: src/components/Pages/RequestEpisode.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from 'react';
-import Card from '../Cards/Card';
+import Card fromhttps://github.com/Avdhesh-Varshney/Chanakya-Niti/pull/22 '../Cards/Card';
```
Comment (Avdhesh-Varshney) [inline src/components/Pages/RequestEpisode.jsx:2]: what's the purpose of these comments and links addition?

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Avdhesh-Varshney** [comment]
@suryapratapsingh26 are you working on it?

**Comment by suryapratapsingh26** [comment]
> @suryapratapsingh26 are you working on it?

I could not understand what's the issue with my pr can you please describe I have resolved the conflict with the base branch  or do I need to create new pr and close this one

File: src/components/Pages/RequestEpisode.jsx (hunk 1 above)
Comment (suryapratapsingh26) [reply src/components/Pages/RequestEpisode.jsx:2]: i have removed these comments and links

**Pull request closed without merging**